### DISCUSSION TITLE
Modify PDF metadata

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1068] [Help]
Can we write information in Pdf metadata? 

Author, Creation Time, Keywords, Creator and any

**Comment by dlemstra** [comment]
You can use the `PdfWriteDefines` to set some of these settings.

**Reply by jaka1-newbie** [reply]
Great! 
How to find all options available by this class?
I found only Title, Author, Producer but in pdf documnet see more: Creator, Keywords, and some more.

**Reply by dlemstra** [reply]
Those are the only ones that are available at this point in time. If you need more it might be a good idea to attach a `.pdf` file here that contains that information. Or a pointer to the spec for adding this. You will probably need to zip the file to upload it here.

**Comment by jaka1-newbie** [comment]
[1.pdf](https://github.com/dlemstra/Magick.NET/files/7487966/1.pdf)

**Comment by dlemstra** [comment]
I meant an example that has those extra fields.

**Comment by jaka1-newbie** [comment]
It has
![Screenshot_2](https://user-images.githubusercontent.com/67605008/140569853-efac38ed-821c-4f44-a14e-5c511e451efa.png)

**Reply by dlemstra** [reply]
I have no idea what you are trying to tell me?
